apps/dashboard/src/@/types/analytics.ts

```diff
@@ -45,6 +45,12 @@ export interface RpcMethodStats {
   count: number;
 }
 
+export interface RpcUsageTypeStats {
```

Comment:
nit: please move this to the file where the function that returns this data is defined